test/acceptance/dashboard_test.clj

```diff
@@ -75,23 +72,21 @@
   (t/testing "When the current user creates a new Haggadah and goes back to the dashboard, the Haggadah is listed first among the Haggadot and a new Haggadah with the same details is added to firestore"
     (doto driver
       (c/home->dashboard)
-      (create-haggadah new-haggadah-title)
-      (e/screenshot "screenshots/create-haggadah-test-before-getting-haggadot.png"))
+      (create-haggadah new-haggadah-title))
+    (e/wait 5)
```

Comment:
Maybe we could wait for something to appear instead? And change the default timeout if necessary

Reply:
I'll change it so it waits for the Haggadot to appear before continuing on.